src/controllers/api_user_controller.py

```diff
@@ -79,21 +97,151 @@ def inser_user():
 
 # UPDATE USER
 def update_user(user_id):
-    user = User.query.get_or_404(user_id)
-    data = request.get_json()
-
-    user.username = data['username']
-    user.full_name = data['full_name']
-    user.email = data['email']
-    user.phone_number = data['phone_number']
-    user.report_access = data['report_access']
-    user.view_costs = data['view_costs']
-    user.last_login_date = data['last_login_date']
-    user.enabled = data['enabled']
-    user.is_corporated = data['is_corporated']
-    user.created_on = data['created_on']
-    user.modified_on = data['modified_on']
-
-    db.session.commit()
-
-    return jsonify({'message': 'User updated successfully'})
+    try:
+        user = User.query.get_or_404(user_id)
+        if user:
+            data = request.get_json()  # data to json
+
+            user.username = data['username']
+            user.full_name = data['full_name']
+            user.email = data['email']
+            user.phone_number = data['phone_number']
+            user.report_access = data['report_access']
+            user.view_costs = data['view_costs']
+            user.last_login_date = data['last_login_date']
+            user.enabled = data['enabled']
+            user.is_corporated = data['is_corporated']
+            user.created_on = data['created_on']
+            user.modified_on = data['modified_on']
+
+            db.session.commit()
+
+            results = {
+                "status_code": 200,
+                "success": True,
+                'message': 'User updated successfully'
+            }
+    except Exception as e:
+        results = {
+            "status_code": 500,
+            "success": False,
+            "message": str(e)
+        }
+    return jsonify(results)
+
+### ------API get data using nativeQuery------------------------------------------------------########
+
+
+# GET LIST USER
+def get_list_native_query():
+    try:
+        query = text(
+            "SELECT column_name FROM information_schema.columns WHERE table_name = 'user' AND table_schema = 'sampleapp'")
```

Comment:
What is the table_schema = 'sampleapp'?
With other env, example production env has database name is "ABC", staging env is "ABC_staging", what happen?

Reply:
- sampleapp: name of my database.
- With other env, I will pass value in this, not hardcode.